HIP/hip-991.md

```diff
@@ -232,6 +215,20 @@ message ConsensusCustomFeeList {
 }
 ```
 
+#### ConsensusSubmitMessageTransactionBody
+
+The `ConsensusSubmitMessageTransactionBody` message is updated to include the optional `max_custom_fees` property for specifying the maximum fee that the user is willing to pay for the message.
```

Comment:
Do we need to call out that there's a method to set accepts all fees in this section?

Reply:
Added the flag `accept_all_fees`

Reply:
It seems that the user must set either a `max_custom_fees` or set the `accept_all_custom_fees` to **true**. We can add a validation to throw an error if `accept_all_custom_fees=false` and `max_custom_fees` is not set.

Reply:
I don't think this is needed. If a topic has custom fees and neither `max_custom_fees` nor `accept_all_custom_fees` are set, the message will fail anyway because the fees cannot be paid.